aiida_registry/fetch_metadata.py

```diff
@@ -244,9 +294,10 @@ def fetch_metadata(filter_list=None, fetch_pypi=True, fetch_pypi_wheel=True):
         plugins_metadata[plugin_name] = complete_plugin_data(
             plugin_data, fetch_pypi=fetch_pypi, fetch_pypi_wheel=fetch_pypi_wheel
         )
+    plugins_metadata = dict(sorted(plugins_metadata.items(), key=lambda x: x[1]['commits_count'], reverse=True))
```

Comment:
Since we are going to commit this to git, I don't think we'll want to apply the sort order here (the order will change over time, leading to unnecessary diffs).

We'll apply the order dynamically in the app.